packages/amplify-graphql-function-transformer/src/__tests__/__snapshots__/amplify-graphql-function-transformer-override.test.ts.snap

```diff
@@ -88,24 +88,52 @@ Object {
             Object {
               "Action": "lambda:InvokeFunction",
               "Effect": "Allow",
-              "Resource": Object {
-                "Fn::If": Array [
-                  "HasEnvironmentParameter",
-                  Object {
-                    "Fn::Sub": Array [
-                      "arn:aws:lambda:\${AWS::Region}:\${AWS::AccountId}:function:echofunction-\${env}",
-                      Object {
-                        "env": Object {
-                          "Ref": "referencetotransformerrootstackenv10C5A902Ref",
+              "Resource": Array [
+                Object {
```

Comment:
See https://github.com/aws-amplify/amplify-category-api/pull/869/files#r992662669 for explanation of this change.